sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/DefaultAvailabilityStrategy.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * The type Availability strategy.
+ */
+public class DefaultAvailabilityStrategy extends AvailabilityStrategy {
```

Comment:
should this be in implementation package? why customer would ever need to configure this DefaultAvailabilityStrategy?

Reply:
They can set the availability strategy do Default if they want. It is the default behavior though!

Reply:
removed this as not needed anymore